card/card.gemspec

```diff
@@ -3,6 +3,7 @@
 require "../decko_gem"
 
 DeckoGem.gem "card" do |s, d|
+  s.required_ruby_version = ">= 3.0.0"
```

Comment:
We should get rid of these and fix CodeClimate.  DeckoGem handles this, and imo it's preferable that we only maintain this in one place.

Reply:
(by "fix" I mean tell it to exclude that check)